packages/components/src/styles/components/time.scss

```
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TIME COMPONENT
+//
+
+.hds-time-range {
```

as mentioned in a comment above
```suggestion
.hds-time--range {
```

---

Fixed. Thanks